spell_check.py

```diff
@@ -0,0 +1,88 @@
+#!/bin/python
```

Comment:
I would change the shebang line to python3

Reply:
Ok